File: Day15/Solution.java
public static  Node insert(Node head,int data)
	{
  	//Complete this method
        Node n = new Node(data);
        Node temp=head;
        if(head==null)
           return n;
        else{
            while(temp.next !=null)
                temp = temp.next;
        }
        temp.next=n;
        return head;
	}

File: Day15/Solution.cpp
Node* insert(Node *head,int data)
      {
         Node* new_node;
          new_node = new Node(data);
          if (head == NULL){
              head = new_node;
          }
          else {
              Node* temp;
              temp = head;
              while (temp->next!=NULL){
                  temp = temp->next;
              }
              temp->next = new_node;
          }
          
          return head;
      }
